src/PanoViewer/PanoViewer.js

```diff
@@ -106,49 +118,95 @@ export default class PanoViewer extends Component {
 	}
 
 	/**
-	 * @typedef {Object} ImageInfo
-	 * @property {String|Image} image Input image url or Image object or Image element <ko>입력 이미지 URL 이나 이미지 객체 나 엘리먼트</ko>
-	 * @property {String} imageType The type of the image: equirectangular, vertival_cubestrip <ko>이미지 유형 : equirectangular, vertival_cubestrip</ko>
+		* Getting the video element that the viewer is currently playing. You can use this for playback.
+		* @ko 뷰어가 현재 사용 중인 비디오 요소를 얻습니다. 이 요소를 이용해 비디오의 컨트롤을 할 수 있습니다.
+		* @method eg.view360.PanoViewer#getVideo
+		* @return {HTMLVideoElement} HTMLVideoElement<ko>HTMLVideoElement</ko>
+		*/
+	getVideo() {
+		if (!this._isVideo || !this._photoSphereRenderer.isImageLoaded()) {
+			return null;
+		}
+
+		return this._photoSphereRenderer.getContent();
+	}
+
+	/**
+	 * Setting the video information to be used by the viewer.
+	 * @ko 뷰어가 사용할 이미지 정보를 설정 합니다.
+	 * @method eg.view360.PanoViewer#setVideo
+	 * @param {String|HTMLVideoElement} video Video URL or Video Tag<ko>비디오 URL 혹은 비디오 태그</ko>
+	 * @param {Object} param
+	 * @param {String} [param.projectionType="equirectangular"] Projection Type<ko>프로젝션 타입</ko>
+	 *
+	 * @return {PanoViewer} PanoViewer instance<ko>PanoViewer 인스턴스</ko>
 	 */
+	setVideo(video, param = {projectionType: PanoImageRenderer.ImageType.EQUIRECTANGULAR}) {
+		if (!video) {
+			return this;
+		}
+
+		this.setImage(video, {projectionType: param.projectionType, isVideo: true});
+		return this;
+	}
 
 	/**
 	 * Getting the image information that the viewer is currently using.
 	 * @ko 뷰어가 현재 사용하고있는 이미지 정보를 얻습니다.
 	 * @method eg.view360.PanoViewer#getImage
-	 * @return {ImageInfo}
+	 * @return {Image} Image Object<ko>이미지 객체</ko>
 	 */
 	getImage() {
-		return {
-			image: this._image,
-			imageType: this._imageType
-		};
+		if (this._isVideo || !this._photoSphereRenderer.isImageLoaded()) {
+			return null;
+		}
+
+		return this._photoSphereRenderer.getContent();
 	}
 
 	/**
 	 * Setting the image information to be used by the viewer.
 	 * @ko 뷰어가 사용할 이미지 정보를 설정 합니다.
 	 * @method eg.view360.PanoViewer#setImage
-	 * @param {ImageInfo} imageInfo Input image information  <ko>입력 이미지 정보</ko>
+	 * @param {String|Image} image ImageURL or Image Object<ko>이미지 URL 혹은 Image 객체</ko>
+	 * @param {Object} param Additional information<ko>이미지 추가 정보</ko>
+	 * @param {String} [param.projectionType="equirectangular"] Projection Type<ko>프로젝션 타입</ko>
+	 *
+	 * @return {PanoViewer} PanoViewer instance<ko>PanoViewer 인스턴스</ko>
 	 */
-	setImage({image, imageType}) {
-		if (image) {
-			this._image = image;
+	setImage(image, param = {
+		projectionType: PanoImageRenderer.ImageType.EQUIRECTANGULAR,
+		isVideo: false
+	}) {
+		const projectionType = param.projectionType || PanoImageRenderer.ImageType.EQUIRECTANGULAR;
+		const isVideo = param.isVideo || false;
+
+		if (this._image && isVideo !== this._isVideo) {
+			console.warn("Not support changing content type(Image <--> Video)");
+			return this;
+		}
```

Comment:
@happyhj I'm begging your opinion for this. 

In above code, if content type is differed then it will do nothing. I consider following way if `set` fails.

  1. throw
  2. return false (if fails)
  3. console
  4. ignore

But each have each problem.
  1. User should try/catch
  2. Different `set` interface (others returns this)
  3. User may not recognize it.
  4. User cannot judge what is the problem.

Reply:
**console.warn** would be nice.
And it will be better to show this is temporary state with warning message.
> "Currently not supporting changing content type(Image <--> Video)"